charts/consul/test/docker/Test.dockerfile

```diff
@@ -6,7 +6,7 @@
 # a script to configure kubectl, potentially install Helm, and run the tests
 # manually. This image only has the dependencies pre-installed.
 
-FROM circleci/golang:1.17
+FROM circleci/golang:1.18
```

Comment:
We need to build and upload a new docker image

Reply:
Are there instructions for this somewhere?

Reply:
I'm not sure where the instructions are, you could try asking ashwin, he did the last bump of our helm test dockerfile: https://github.com/hashicorp/consul-k8s/pull/1177/files